assets/src/domain/eventEditor/ui/tickets/TicketRegistrationsLink.tsx

```diff
@@ -26,19 +27,22 @@ const TicketRegistrationsLink: React.FC<Props> = ({ ticket }) => {
 		ticket_id: ticket.dbId,
 		return: 'edit',
 	});
-	const title = __('view registrations for this ticket.');
+	const buttonTitle = __('view registrations for this ticket.');
+	const countTitle = __('total registrations.');
 
 	return (
-		<Tooltip title={title}>
-			<a
-				href={regListUrl}
-				className={'ee-editor-details-reg-url-link'}
-				target={'_blank'}
-				rel={'noopener norefferer'}
-			>
-				<EspressoIcon icon={Icon.GROUPS} svgSize={24} />
-			</a>
-		</Tooltip>
+		<ItemCount count={ticket.registrationCount} title={countTitle} zeroCountChar={'0'} emphasizeZero={false}>
```

Comment:
let's switch default to `false`, so when we'll have to enable it we will pass just like this, this way we will make props more succint:
```
<ItemCount count={ticket.registrationCount} title={countTitle} zeroCountChar={'0'} emphasizeZero>
```